contracts/ligo/src/fa2_swap.mligo

```diff
@@ -0,0 +1,181 @@
+(* Swaps contract.
+Specification: https://hackmd.io/iArxW3FyR_e85JNPYvuQhQ.
```

Comment:
Add REAMME.md file into `minter-sdk` repo instead

Reply:
Oh right, I missed that all the contracts are described there :sweat_smile: